qcodes/tests/test_slack.py

```diff
@@ -1,28 +1,100 @@
-def test_slack(mocker):
+import pytest
 
+import qcodes.utils.slack
```

Comment:
to prevent the need to slack_sdk to be installed for these tests to run, i suggest to use the same trick as the old tests used - `mocker.patch.dict('sys.modules', slack_sdk=mock_slack_sdk)`. it can be implemented as a autouse=True pytest fixture so that it automatically applies to all tests in this module

Reply:
Yes, I did something similar. There was only one test in there so far, or am I missing something?

Reply:
> am I missing something?

the test module explicitly imports qcodes.utils.slack, hence importing slack_sdk which is not installed during CI. so removing the import of qcodes.utils.slack, or doing the import AFTER patching the slack_sdk module, should help

Reply:
Aha, I was wondering why the import was not at the top. I'll put patching the module back in